fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/TabItemTokens.kt

```diff
@@ -158,10 +165,15 @@ open class TabItemTokens : IControlToken, Parcelable {
 
     @Composable
     open fun padding(tabItemInfo: TabItemInfo): PaddingValues {
-        return when(tabItemInfo.tabTextAlignment){
+        return when (tabItemInfo.tabTextAlignment) {
             TabTextAlignment.HORIZONTAL -> PaddingValues(top = 8.dp, start = 4.dp, bottom = 4.dp, end = 8.dp)
             TabTextAlignment.VERTICAL -> PaddingValues(top = 8.dp, start = 8.dp, bottom = 4.dp, end = 8.dp)
             TabTextAlignment.NO_TEXT -> PaddingValues(top = 8.dp, start = 8.dp, bottom = 4.dp, end = 8.dp)
         }
     }
+
+    @Composable
+    open fun textSize(tabItemInfo: TabItemInfo): TextUnit {
```

Comment:
Instead of textsize create a function to return typograpghy for the coherence among API usage.

Refer this:
https://github.com/microsoft/fluentui-android/blob/01664c5999ff05c8c8b61be2afca758a7088a3a9/fluentui_core/src/main/java/com/microsoft/fluentui/theme/token/controlTokens/BannerTokens.kt#L67